### ISSUE TITLE
A Transfer Learning Framework for Anomaly Detection Using Model of Normality. (arXiv:2011.06210v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#936]
A Transfer Learning Framework for Anomaly Detection Using Model of Normality. (arXiv:2011.06210v1 [cs.LG])<br>
https://ift.tt/3lu33yZ<br>
<p>Convolutional Neural Network (CNN) techniques have proven to be very useful in image-based anomaly detection applications. CNN can be used as deep features extractor where other anomaly detection techniques are applied on these features. For this scenario, using transfer learning is common since pretrained models provide deep feature representations that are useful for anomaly detection tasks. Consequentially, anomaly can be detected by applying similarly measure between extracted features and a defined model of normality. A key factor in such approaches is the decision threshold used for detecting anomaly. While most of the proposed methods focus on the approach itself, slight attention has been paid to address decision threshold settings. In this paper, we tackle this problem and propose a welldefined method to set the working-point decision threshold that improves detection accuracy. We introduce a transfer learning framework for anomaly detection based on similarity measure with a Model of Normality (MoN) and show that with the proposed threshold settings, a significant performance improvement can be achieved. Moreover, the framework has low complexity with relaxed computational requirements.</p>
<br>
<br>
via cs.CV updates on arXiv.org http://arxiv.org/<br>
